### ISSUE TITLE
EnergyPlus 23.2.0

### ISSUE THREAD

**Issue by RVerschoren** [vscentrum/vsc-software-stack#228]
* link to support ticket: 2885 (UAntwerp)
* website: https://energyplus.net
* installation docs: https://github.com/NREL/EnergyPlus/wiki/Building-EnergyPlus
* toolchain: `foss/2022a`
* easyblock to use: `CMakeMake`
* required dependencies:
  * [x] git
  * [x] CMake
* optional dependencies:
* notes:
  * ... 
* effort: low
* other install methods
  * conda: no
  * container image: no
  * pre-built binaries (RHEL8 Linux x86_64): no
  * easyconfig outside EasyBuild: yes (tarball of version 9.3.0)

**Comment by RVerschoren** [comment]
Uploaded related EasyConfigs to `wip/228_EnergyPlus/`, for versions 9.6.0 and 23.2.0.

@boegel These could both be submitted to EasyBuild, but I don't suppose 9.6.0 (released in 2021) is still relevant...
Only submitted [23.2.0](https://github.com/easybuilders/easybuild-easyconfigs/pull/19565) for now.

**Comment by boegel** [comment]
@RVerschoren Do we still need to keep this open?

**Comment by RVerschoren** [comment]
> [@RVerschoren](https://github.com/RVerschoren) Do we still need to keep this open?

No, the easyconfig was accepted in easybuilders/easybuild-easyconfigs#19565, so this issue can indeed be closed!
